helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -122,7 +124,7 @@
   // TODO: make this be per _manager or per _listener instaed of per callbackHandler -- Lei
   private CallbackProcessor _batchCallbackProcessor;
   private boolean _watchChild = true; // Whether we should subscribe to the child znode's data
-                                      // change.
+  // change.
```

Comment:
Maybe no need to format the whole file? Or you can move this comment before the code:
// comment
code;

Reply:
This is because the previous code was not correctly formatted in the switch section.